### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by Nizamuddin4493**
- Used JavaScript classes instead of objects and arrays.
- Created class methods to add and remove books.
- Maintained all of the original functionality of the application.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1,27 +1,136 @@
+/* stylelint-disable selector-class-pattern */
+/* stylelint-disable shorthand-property-no-redundant-values */
```
Comment (brytebee) [inline css/styles.css]: - [ ] **Required:** It is not a good practice to disable the linters. This enforces uniformity of code within the Microverse community and the workplace in future. Kindly fix the linter errors.

File: js/classes.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+const myForm = document.querySelector('#myForm');
+resetForm.addEventListener('click', () => {
+  myForm.reset();
+});
+
+const list = document.querySelector('.list-action');
+const add = document.querySelector('.add-action');
+const contact = document.querySelector('.contact-action');
+
+list.addEventListener('click', () => {
```
Comment (brytebee) [inline js/classes.js]: - [ ] **Required:** Kindly fix all console errors. See picture below:

image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @fernando-silvabr66 & @Nizamuddin4493👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- descriptive commit messages✔️
- excellent class implementation✔️
- clean code✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — APPROVED** [review]
Hi @Nizamuddin4493,


## Wow, you made it 🎉
Brilliant
## Thank you for implementing the changes requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**